src/Agent.Worker/ExecutionContext.cs

```diff
@@ -439,6 +446,15 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
                 }
             }
 
+            // Runtime option variables
```

Comment:
what is this for? why is this here?

Reply:
the git source provide needs to know whether to use SChanel or OpenSSl as secure backend

Reply:
k, lets chat offline. i'm still confused why the variable is set here